pkg/actors/actors.go

```diff
@@ -372,7 +372,7 @@ func (a *actorsRuntime) TransactionalStateOperation(ctx context.Context, req *Tr
 	if a.store == nil {
 		return errors.New("actors: state store does not exist or incorrectly configured")
 	}
-	requests := []state.TransactionalRequest{}
+	operations := []state.TransactionalStateOperation{}
```

Comment:
@LMWF Do you want to allow users to set metadata for transactional api? it will be breaking change.

Reply:
our sdks are not passing it, and for actors I don't foresee passing a state store specific metadata.

Reply:
@amanbha Thanks for the confirmation. Once @LMWF reviewed, I will merge it.